popup.js

```diff
@@ -1,38 +1,52 @@
-let active = document.getElementById('active');
+const active = document.getElementById('active');
+const reloadInput = document.getElementById('reload-int-input');
 
+// Maintain checked input box if this window has active reload
 chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
   let tab = tabs[0];
   chrome.storage.local.get(tab.id.toString(), (items) => {
-      if(items[tab.id]){
-          console.log("found!");
-          active.checked = true;
-          
-      }
+    // Load reload interval if previously inputted. Otherwise, default value is 10 seconds
+    if (items[tab.id]) {
+      reloadInput.value = items[tab.id].input ? items[tab.id].input : 10;
```

Comment:
Tested, this does not work for the first time. When I open the input box for the first time, it shows me a blank input box. But yeah once I activate it, and then any state change shows 10, which is good.

Reply:
This needs a fix I guess.

Reply:
Sorry, displaying the 10 when the popup is first loaded totally slipped my mind. Do you think it would be alright to just set value='10' in the element?